### PR TITLE
Add Git Hooks

### PR DESCRIPTION
**Pull request by juli1**
### What does this PR do? What is the motivation?

This PR contains information to use Git Hooks for the Code Analysis product.

### CODE REVIEW

File: content/en/code_analysis/git_hooks/_index.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+---
+title: Code Analysis Git Hooks
```
Comment (urseberry) [inline content/en/code_analysis/git_hooks/_index.md:2]:
```suggestion
title: Git Hooks
```

File: content/en/code_analysis/git_hooks/_index.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+description: Prevent code merge of errors
```
Comment (urseberry) [inline content/en/code_analysis/git_hooks/_index.md:3]:
```suggestion
description: Prevent the merging of code with errors
```

File: content/en/code_analysis/git_hooks/_index.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  tag: "Documentation"
+  text: "Learn about Static Analysis"
+- link: "/code_analysis/software_composition_analysis/"
+  tag: "Documentation"
+  text: "Learn about Software Composition Analysis"
+---
+
+## What is a Git hook
+
+A [Git Hook](https://git-scm.com/docs/githooks) is a program executed before a user commits code on a repository
```
Comment (urseberry) [inline content/en/code_analysis/git_hooks/_index.md:18]:
```suggestion
A [Git hook](https://git-scm.com/docs/githooks) is a program executed before a user commits code on a repository
```

**Review by urseberry — COMMENTED** [review]
Please reformat all of your links to use numbered links: https://datadoghq.atlassian.net/wiki/spaces/docs4docs/pages/1182958054/Markdown+Shortcode+Reference#Links. The translations system doesn't support inline links.

File: content/en/code_analysis/git_hooks/_index.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    exit 1
+fi
+```
+
+Note that the program accepts the following parameters:
+
+ - `--confirmation`: ask the user for confirmation to override the Git Hook check
+ - `--default-branch`: specify the name of the default branch
+ - `--secrets`: enable secrets detection (in private beta)
+ - `--output <file>`: export the findings found in the commit into a SARIF file
```
Comment (urseberry) [inline content/en/code_analysis/git_hooks/_index.md:69]:
```suggestion
 - `--confirmation`: Ask the user for confirmation to override the Git hook check
 - `--default-branch`: Specify the name of the default branch.
 - `--secrets`: Enable secrets detection (private beta).
 - `--output <file>`: Export the findings found in the commit into a SARIF file.
```

**Review by urseberry — APPROVED** [review]

**Comment by urseberry** [comment]
Is this ready to merge? If not, please post in #documentation in Slack when you are ready for it to be merged.

**Review by dastrong — APPROVED** [review]

**Comment by juli1** [comment]
> Is this ready to merge? If not, please post in #documentation in Slack when you are ready for it to be merged.

Ready to merge!

**Pull request merged**